CONTRIBUTING.md

```diff
@@ -52,6 +52,37 @@ All contributed code, docs, and other materials are considered licensed under th
   ```bash
   sbt doc
   ```
+  
+## Logging Documentation
+
+- We use the [Spark Logging interface](https://spark.apache.org/docs/1.5.2/api/java/org/apache/spark/Logging.html#:~:text=Interface%20Logging&text=Creates%20a%20SLF4J%20logger%20for,intended%20as%20an%20internal%20utility).
+- Spark uses `log4j` for logging. You can configure it by adding a `log4j2.properties` file in the conf directory. One way to start is to copy the existing `log4j2.properties.template` located there.
+
+An example of using logging on a class is:
+
+```scala
+import org.apache.spark.internal.Logging
+
+case class MyClass() extends Logging {
+
+  def myMethod(): Unit = {
+    logInfo("This is an info message")
+    logWarn("This is a warning message")
+    logError("This is an error message")
+    logTrace("This is a trace message")
+    logDebug("This is a debug message")
+  }
+}
+
+```
+
+The following log levels are used to track code behaviour:
```

Comment:
It would be good to have examples that developers can follow. The cheap way to do it is to say that they should search the code for examples.

Reply:
I will add a few scala lines 🐒

Reply:
Done in 32bae1a2